issuing_using_did.md

```diff
@@ -546,35 +222,96 @@ The `DID` type will simply allow for:
+  "signature": {
+    "type": "SHA3MerkleProof",
+    "targetHash": "0ff2bee4bc5e0cde00dbf4524ca5173724538e3bf46241ff653e07ae51141c4d",
+    "proof": [],
+    "merkleRoot": "0ff2bee4bc5e0cde00dbf4524ca5173724538e3bf46241ff653e07ae51141c4d"
+  },
   "proof": [
     {
-      "type": "DidGenericSignature",
+      "type": "OpenAttestationSignature2018",
+      "created": "2020-10-05T09:05:35.171Z",
       "proofPurpose": "assertionMethod",
-      "signature": "<signed merkle root>"
+      "verificationMethod": "did:ethr:0x6813Eb9362372EEF6200f3b1dbC3f819671cBA69#controller",
+      "signature": "0x6d0ff5c64b8230cdc471f38267495002f2c762acf7a80250599809ee32b4255377f1adcb56fb712dee66bfeb21be6b5d802f299aea1f1edca129e88e4c1742ce1c"
     }
   ]
 }
 ```
+
+## Implementation Progress
+
+### Done
+
+#### Upgrade @govtechsg/open-attestation
+
+[#127](https://github.com/Open-Attestation/open-attestation/pull/127)
+
+Schema has been updated to allow `DID` and `DNS-DID` type for `identityProof`.
+Released as v4.0.0.
+
+#### Upgrade @govtechsg/oa-verify
+
+[#114](https://github.com/Open-Attestation/oa-verify/pull/114)
+
+Added `OpenAttestationDidSignedDocumentStatus` fragment to verify document signed directly.
+Added `OpenAttestationDnsDid` fragment to verify document with top level DNS identifier, but signing with DID
+Added `OpenAttestationDidSignedDidIdentityProof` fragment to verify document with top level DID identifier, and signing with DID
+Added all except `OpenAttestationDidSignedDidIdentityProof` into the default set of verifiers.
+Released as v5.1.0
+
+#### Upgrade @govtechsg/open-attestation-cli
+
+[#93](https://github.com/Open-Attestation/open-attestation-cli/pull/93)
+
+Added command to sign wrapped documents with DID
+Released as v1.29.0
+
+### In Process
+
+#### Upgrade TradeTrust website to support new verification
+
+[#249](https://github.com/TradeTrust/tradetrust-website/pull/249)
+
+Allow documents signed with `DNS-DID` to be verified
+
+### To Be Done
+
+#### Revocation method: Revocation Store
+
+Implement the schema, oa-verify method to allow a document store to be used as a revocation store for documents which can be revoked, instead of just using `NONE` in the `revocation.type`.
+
+#### Individual document wrapping in CLI
+
+Allow documents to be wrapped individually instead of batching it up. This prevent accidental issuance when two documents share the same merkle root and the proof is copied over from one document to another when one document has not been appended the proof.
+
+#### IO Concurrency in CLI
+
+Optimize IO concurrency in CLI by allow a limited number of threads to run concurrently. Current implementation does maximum parallelisation using `Promise.all`.
+
+#### Counterfactually deploying document store as revocation store
+
+This allow a user to "reserve" an address for the revocation store in the future but not perform the action now. The user will still be able to use the address during the document creation phase.
```

Comment:
Regarding this, I was wondering if there could be an issue for the verifier. Assuming we reserve an address for the revocation store and add the address into the file, will the verifier able to identify correctly that the contract has not been deployed ? How will we differentiate with wrong addresses (human mistake) ?